.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,46 @@
+# Proposed change
+
+<!--- 
+Please include a summary of the change and which issue is fixed in the form of fix #XX for auto-linking.
+
+- Include relevant motivation and context
```

Comment:
Wouldn't the relevant motivation and context be in the linked issue? Not sure of the value of repeating it in the PR as well.